mowgli_etl/pipeline/wdc/parsimonious_parser/wdc_parsimonious_node_visitor.py

```diff
@@ -0,0 +1,26 @@
+from parsimonious.nodes import NodeVisitor
+
+
+class WdcParsimoniousNodeVisitor(NodeVisitor):
+    def __init__(self):
+        self.dictionary = {}
+
+    def visit_dimension(self, node, visited_children):
+        entries = node.text.split(" ")
+        value = ".".join(entries[0:-1])
+        key = entries[-1]
+        if key == "l":
```

Comment:
Use a map for direct string-string translation, + themap.get for the missing case.